lightning_pose/utils/predictions.py

```diff
@@ -366,7 +366,7 @@ def _predict_frames(
                     image = batch["images"].to("cuda:%i" % gpu_id)
             else:
                 image = batch  # predicting from video
-            outputs = model.forward(image)
+            outputs = model.forward(image, mode='2d_context')
```

Comment:
not sure about this `mode="2d_context"` now

Reply:
Yes, removed. Thanks!